.travis.yml

```diff
@@ -115,6 +115,14 @@ install:
   - python setup.py build_ext --inplace
   - python setup.py develop
 
+before_script:
+  - pip install flake8
+  - echo -n "flake8 testing of ${URL} on " ; python -V
+  # stop the build if there are Python syntax errors or undefined names
+  - flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics
```

Comment:
Could some of this go in a setup.cfg file?  That's the direction we should be moving in medium-term anyway.